datacapture/src/test/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDatePickerViewHolderFactoryTest.kt

```diff
@@ -133,6 +133,67 @@ class QuestionnaireItemDatePickerViewHolderFactoryTest {
       .isEqualTo("11/19/20")
   }
 
+  @Test
+  fun `parse date text input as per US locale`() {
```

Comment:
nit:

```suggestion
  fun `parse date text input in US locale`() {
```

and below